src/NGen.cpp

```diff
@@ -57,7 +58,93 @@ int mpi_num_procs;
 
 std::unordered_map<std::string, std::ofstream> nexus_outfiles;
 
+void ngen::exec_info::runtime_summary(std::ostream& stream) noexcept
+{
+    stream << "Runtime configuration summary:\n";
+
+#if NGEN_WITH_MPI
+
+    MPI_Init(nullptr, nullptr);
```

Comment:
Could you move this call to `MPI_Init` down to the little block of code added to `main()`? Both for symmetry, and so that `runtime_summary` could potentially be called even in a run of ngen that's actually going to use MPI

Reply:
Added in [6734ba2](https://github.com/NOAA-OWP/ngen/pull/679/commits/6734ba213913c4da078fe63141253cc1b88eebc5).